book/02-git-basics/sections/recording-changes.asc

```diff
@@ -204,7 +204,7 @@ $ cat .gitignore
 ----
 
 The first line tells Git to ignore any files ending in ``.o'' or ``.a'' – object and archive files that may be the product of building your code.
-The second line tells Git to ignore all files that end with a tilde (`~`), which is used by many text editors such as Emacs to mark temporary files.
+The second line tells Git to ignore all files that end with a tilde (`~`) in the filename, which is used by many text editors such as Emacs to mark temporary files.
```

Comment:
ok... i changed and pushed...  you think 

If your preferred editor is Emacs or Vim, you know about files that end with a `~` or `.swp` in the filename.

will be

If your preferred editor is Emacs or Vim, you know about files whose names end with a `~` or `.swp`.

and 
The second line tells Git to ignore all files that end with a tilde (`~`) in the filename, which is used by many text editors such as Emacs to mark temporary files.

to 

The second line tells Git to ignore all files whose names end with a tilde (`~`), which is used by many text editors such as Emacs to mark temporary files.

may sound repetitive, but they are in different sections of the book.